client/my-sites/stats/activity-log/index.jsx

```diff
@@ -604,14 +602,15 @@ class ActivityLog extends Component {
 				{ hasFirstBackup && this.renderMonthNavigation() }
 				{ this.renderActionProgress() }
 				{ ! isRewindActive && !! isPressable && <ActivityLogRewindToggle siteId={ siteId } /> }
-				{ isNull( logs ) && (
-					<section className="activity-log__wrapper">
-						<ActivityLogDayPlaceholder />
-						<ActivityLogDayPlaceholder />
-						<ActivityLogDayPlaceholder />
-					</section>
-				) }
-				{ ! isNull( logs ) &&
+				{ requestData.logs.isLoading &&
```

Comment:
can you just check against hasLoaded?

Reply:
yes! I think so - simplified in 11520a62e4